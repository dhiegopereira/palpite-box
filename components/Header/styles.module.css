.wrapper {
    @apply bg-gray-200 p-4  shadow-md;
}
.img {
    @apply container mx-auto;
}
.logo {
    @apply mx-auto;
}
.menu {
    @apply bg-gray-300 p-4  shadow-md text-center;
}
.menu_link {
    @apply px-2 hover:underline;
}