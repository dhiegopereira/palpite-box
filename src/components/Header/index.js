import React from 'react'
import Link from 'next/link'

const Header = () => {
    return (
        <React.Fragment>
        <div className='bg-gray-200 p-4  shadow-md'>
            <div className='container mx-auto'>
                <Link href='/'>
                    <a><img className='mx-auto' src='/img/logo_palpitebox.png' alt='PalpiteBox' /></a>
                </Link>
            </div>
        </div>        
        </React.Fragment>
    )
}

export default Header