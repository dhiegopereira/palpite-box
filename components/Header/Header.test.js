import React from 'react'
import {render, screen} from '@testing-library/react'
import renderer from 'react-test-renderer';
import Header from '.'

describe('Header Component', () => {
    it('Search site logo', () => {
        render(<Header/>)
        screen.getByRole('img', {name: /PalpiteBox/i})
    })

    it('Snapshot Header', () => {
        const tree = renderer
            .create(<Header/>)
            .toJSON();
        expect(tree).toMatchSnapshot();        
    })
})