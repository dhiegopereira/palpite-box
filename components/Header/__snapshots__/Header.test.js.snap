// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Snapshot Header 1`] = `
<div
  className="bg-gray-200 p-4  shadow-md"
>
  <div
    className="container mx-auto"
  >
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <img
        alt="PalpiteBox"
        className="mx-auto"
        src="/img/logo_palpitebox.png"
      />
    </a>
  </div>
</div>
`;
